src/store/mutations.ts

```diff
@@ -373,35 +378,25 @@ export const setErrors = (state: StateIF, errors: string) => {
   state.stateModel.newRequestModel.errors = [errors]
 }
 
-export const setNrResponse = (state: StateIF, data: NameRequestI): boolean => {
+export const setNrResponse = (state: StateIF, nr: NameRequestI) => {
   try {
-    state.stateModel.newRequestModel.nr = data
-    const { applicants = [] } = state.stateModel.newRequestModel.nr
+    // set NR data
+    mutateNameRequest(state, nr)
 
+    // set Applicants
+    const applicants = state.stateModel.newRequestModel.nr?.applicants || []
     if (applicants instanceof Array) {
-      // setApplicantDetails(applicants[0]) // OLD CODE
       state.stateModel.newRequestModel.applicant = { ...applicants[0] }
     } else if (applicants) {
-      // setApplicantDetails(applicants) // OLD CODE
       state.stateModel.newRequestModel.applicant = { ...applicants }
     } else {
       // applicants is null/undefined
     }
-    // TODO: have 1 mutation for nr / applicant info and put these there
```

Comment:
This code is not needed as it it already in `mutateNameRequest` that is now called at the top of this method.